plugin/example/source/dextool/plugin/cpptestdouble/runner.d

```diff
@@ -0,0 +1,150 @@
+/**
+Copyright: Copyright (c) 2017, Joakim Brännström. All rights reserved.
+License: MPL-2
+Author: Joakim Brännström (joakim.brannstrom@gmx.com)
+
+This Source Code Form is subject to the terms of the Mozilla Public License,
+v.2.0. If a copy of the MPL was not distributed with this file, You can obtain
+one at http://mozilla.org/MPL/2.0/.
+
+This file contains an example plugin.
```

Comment:
See #363 .
The plugin solves the meta-problem "an example of a simple plugin do not exist".
It demonstrates how to traverse the AST, how to hook-up to the plugin architecture and some basic architectural principles.

I don't think it is feasible to get the user to think about a problem _it_ wants to solve because it is an undecidable problem. The writer of the description cannot know this.